global-try-spam.sieve

```diff
@@ -0,0 +1,9 @@
+require ["fileinto", "mailbox", "spamtest", "relational", "comparator-i;ascii-numeric"];
+
+# don't create INBOX/Spam; only sort mail if it already exists
+if mailboxexists "INBOX/Spam" {
```

Comment:
tbh ... if you use the default config with the special-use mailboxes the folder should probably exist already.

and the reason why we enforced the sorting here, is to make the "learn as ham" part work ... otherwise one might have to introduce a "ham" mailbox just for training that. currently the whole train has ham workflow works with the admittedly opinionated defaults.

Reply:
This is just an example file and not used in the provided config - I'll keep using it, and just wanted to provide it too, but I don't mind if it's not in the repo.